apps/www/_alternatives/supabase-vs-heroku-postgres.mdx

```diff
@@ -0,0 +1,77 @@
+
+Heroku Postgres and Supabase both offer:
+
+- [A web UI for managing your instance](https://supabase.com/docs/guides/database#table-view)
+- SLAs and Enterprise-grade support packages
+- Direct SSL connections to Postgres
+- Postgres Extensions (see [Supabase Extensions](https://supabase.com/docs/guides/database/extensions), [Heroku Extensions](https://devcenter.heroku.com/articles/heroku-postgres-extensions-postgis-full-text-search))
+- [Backups and PITR](https://supabase.com/blog/2020/08/02/continuous-postgresql-backup-walg) (not on Heroku’s Hobby tier)
+- [Postgres logs](https://supabase.com/docs/guides/platform/logs) (not on Heroku’s Hobby tier)
+- Encryption-at-rest (not on Heroku’s Hobby tier)
+
+## What are the differences?
+
+### Core Features
+
+Both solutions run PostgreSQL, but in a time when Developer Experience matters, there is a lot you can do to improve the speed at which developers can build products faster and with less human resource. These are some of the key differences between Heroku Postgres and Supabase in terms of features:
+
+- Supabase is more than just the raw database, it also comes with:
+  - [Connection pooling](https://supabase.com/docs/guides/database/connecting-to-postgres#connection-pool) so that you won’t run out of connections in a serverless environment.
+  - [Auto-generated APIs](https://supabase.com/docs/guides/api#rest-api-2) based on your schema, so you can communicate with your database directly from the client.
+  - [Realtime API](https://supabase.com/docs/reference/dart/subscribe) is useful for when you want to subscribe to changes to your database over websockets.
+  - [Auth API](https://supabase.com/auth) can be used to leverage Postgres’s Row Level Security model, and control access to sensitive data on a per user, or per group level.
+  - [Functions](https://supabase.com/edge-functions) can be deployed out to the edge directly from the Supabase CLI, which means you can run sensitive business logic or transformations in a serverless fashion.
+  - [File Storage](https://supabase.com/storage) is useful for when your app needs to store large files and folders that aren’t suitable for storing within Postgres itself.
+  - A spreadsheet-like web interface for building your schemas and inspecting data.
+- You can also deploy edge functions to Heroku using their [Dynos](https://www.heroku.com/dynos) runtime in conjunction with something like [Fastly](https://www.fastly.com/).
+
+### Pricing
+
+Simple and predictable pricing are key. The two services price quite differently, the key differences being:
+
+- [Supabase pricing is based around usage](https://supabase.com/pricing), so you only pay for what you use.
+- Heroku prices based on a tier model with [37 plans to choose from](https://elements.heroku.com/addons/heroku-postgresql#pricing).
+
+Supabase’s free tier also includes a dedicated Postgres instance, and the best bit is you can upgrade to pro later without any interruptions.
```

Comment:
> and the best bit is

Also an opinion!

I think the general mindset we should use when writing these is - how would Heroku write this? If they read what we wrote, would they feel hard-done by? If I put my "Heroku" hat on, this feels like it's a Supabase shill